eZ/Bundle/EzPublishElasticsearchSearchEngineBundle/EzPublishElasticsearchSearchEngineBundle.php

```diff
@@ -20,7 +21,7 @@
 use eZ\Publish\Core\Base\Container\Compiler\Search\FieldRegistryPass;
 use eZ\Publish\Core\Base\Container\Compiler\Search\SignalSlotPass;
 
-class EzPublishElasticsearchBundle extends Bundle
+class EzPublishElasticsearchSearchEngineBundle extends Bundle
```

Comment:
Added as suggested to `changes-6.0.md` in https://github.com/ezsystems/ezpublish-kernel/commit/52339375ad55549230dd3443a3512765aa99fc1f.